vscode/README.md

```diff
@@ -1,29 +1,24 @@
 # Q# Language Support for VS Code
 
-This VS Code extension contains:
+_This extension is an unstable product that is still under active development. We do not recommend relying on this in production environments._
```

Comment:
Minor nit, but I think just "This is a development build and may be unstable." is good for the first sentence. (And the second is great as-is).